packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -168,42 +196,44 @@ export class AtomicSearchBox implements AtomicSearchBoxOptions {
     return this.searchBoxState.suggestions.map((suggestion, index) => {
       const id = `${this._id}-suggestion-${index}`;
       return (
-        <button
-          type="button"
-          tabIndex={-1}
+        <li
+          role="option"
           onClick={(e) => this.onClickSuggestion(e)}
           onMouseDown={(e) => e.preventDefault()}
           part="suggestion"
           id={id}
-          value={suggestion.rawValue}
+          class="suggestion h-input px-2 cursor-pointer text-left text-sm bg-transparent border-none shadow-none hover:bg-light-grey flex flex-row items-center"
           innerHTML={suggestion.highlightedValue}
-        ></button>
+          value={index}
+        ></li>
       );
     });
   }
 
   public render() {
     return (
       <div>
-        {this.leadingSubmitButton && this.submitButton}
-
-        <div>
-          <div ref={(el) => (this.containerRef = el as HTMLElement)}>
-            {this.input}
-            {this.clearButton}
-          </div>
-
-          <div>
-            <div
-              part="suggestions"
-              ref={(el) => (this.valuesRef = el as HTMLElement)}
-            >
-              {this.suggestions}
-            </div>
-          </div>
+        <div
+          class="box-border w-full lg:w-80 h-input border border-solid rounded border-medium-grey flex flex-row align-items-center focus:rounded-b-0"
```

Comment:
Something to keep in mind: 

I have a feeling that this combination of classes *might* repeat itself in other places in the project.

For example: Facet search input will share a lot of similarity.

We need to keep track of these repeated pattern and aggressively abstract them in a component that we reuse everywhere when they show up.